lib/models/Character.js

```diff
@@ -16,6 +16,17 @@ class Character {
   }
 
   static async getAll() {
+    const { rows } = await pool.query(
+      `SELECT characters.*, 
+      COALESCE(
+        json_agg(to_jsonb(quotes))
+        FILTER (WHERE quotes.id IS NOT NULL), '[]'
+    ) as quotes from characters 
```

Comment:
Make sure to keep your SQL capitalization consistent - either way(cap, or lowercase) is ok!